src/govuk/template.njk

```diff
@@ -15,11 +18,20 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
 
     {% block headIcons %}
+      {% if _rebrand %}
       <link rel="icon" sizes="48x48" href="{{ assetPath }}/images/favicon.ico">
       <link rel="icon" sizes="any" href="{{ assetPath }}/images/favicon.svg" type="image/svg+xml">
       <link rel="mask-icon" href="{{ assetPath }}/images/govuk-icon-mask.svg" color="{{ themeColor }}">
       <link rel="apple-touch-icon" href="{{ assetPath }}/images/govuk-icon-180.png">
       <link rel="manifest" href="{{ assetPath }}/manifest.json">
+      {% else %}
+      <link rel="shortcut icon" sizes="16x16 32x32 48x48" href="{{ assetPath | default('/assets') }}/images/favicon.ico" type="image/x-icon">
```

Comment:
We know `assetPath` will _always_ be set because of line 9, to avoid confusion can we remove the `default` filters?

Also, it's hard to understand why this is changing as part of this commit – was this just overlooked in #5955? Can we either split into a separate commit or at least mention in the commit message so we have the context.

Reply:
On it not being part of #5955, basically yeah this is my bad. I didn't pick up the difference in assets at review. I'm gonna redo the commit history to be more sensible.